plugins/inputs/radius/radius.go

```diff
@@ -0,0 +1,104 @@
+package radius
```

Comment:
We need the `go:generate` comment to run the readme-tool on `make docs` 
```suggestion
//go:generate ../../../tools/readme_config_includer/generator
package radius
```